connection_manager.go

```diff
@@ -280,22 +274,21 @@ func (n *connectionManager) handleInvalidCertificate(now time.Time, vpnIp iputil
 		return false
 	}
 
-	valid, err := remoteCert.Verify(now, n.intf.caPool)
+	valid, _ := remoteCert.Verify(now, n.intf.caPool)
 	if valid {
 		return false
 	}
 
 	fingerprint, _ := remoteCert.Sha256Sum()
-	n.l.WithField("vpnIp", vpnIp).WithError(err).
-		WithField("certName", remoteCert.Details.Name).
+	hostinfo.logger(n.l).
```

Comment:
This no longer logs the reason the cert was rejected, is that intentional?

Reply:
Nice catch, was not intentional